doc/aurchain.md

```diff
@@ -0,0 +1,23 @@
+# aurchain
+
+## Name
+
+aurchain - Solve and order dependencies using the AUR RPC
+
+## Synopsis
+
+aurchain package [package, ...]
+
+## Description
+
+aurchain takes a list of package names and then solves and orders the dependencies for those packages.
```

Comment:
A list of package names implies it takes a file as an argument, however, it
takes package names as command-line arguments directly.